### ISSUE TITLE
Communication error: 1

### ISSUE THREAD

**Issue by SturmB** [adobe-photoshop/generator-core#422]
I'm following along with Davide Barranca's [Native Applications for Photoshop](https://www.ps-scripting.com/js-native-apps-for-photoshop.html) course and so far everything has been working well once I got NW.js to run on Windows.

I'm now having an issue using his [starter code](https://github.com/undavide/generator-starter) for using Generator. I seem to get one of two error results.

1. An "ECONNRESET" error:

![image](https://user-images.githubusercontent.com/6216224/97195306-91d7ef80-1781-11eb-8ced-5d5aa19742cc.png)

2. An "ECONNABORTED" error:

![image](https://user-images.githubusercontent.com/6216224/97206551-40cef800-178f-11eb-972c-4d690e442ae6.png)

I have tried this with both Photoshop 2020 and Photoshop 2021 with the same result. I can tell that it's _starting_ to communicate with Photoshop because I get a different error (ECONNABORTED) when PS isn't running.

Can anyone please help me to understand why I am getting these errors and, more importantly, how to address them?

**Comment by SturmB** [comment]
Seems that one little [underdocumented hangup](https://github.com/adobe-photoshop/generator-core/wiki/Generator-Development-Environment-Setup#change-generator-settings-in-photoshop) was preventing the app from working. I've [written about it in a blog post](https://chrismcgee.info/why-your-generator-app-isnt-connecting-to-adobe-photoshop/).

TL;DR: Make sure to use "password" for the Password in the _Enable Remote Connections_ setting in Photoshop.
